image-segmentation/custom-recipes/score_image/recipe.py

```diff
@@ -0,0 +1,140 @@
+#############################
+# Your original recipe
+#############################
+
+# -------------------------------------------------------------------------------- NOTEBOOK-CELL: CODE
+# -*- coding: utf-8 -*-
+import dataiku
```

Comment:
that's A LOT of import, do you use them all ?

Reply:
nope, I removed the unnecessary ones